### ISSUE TITLE
Needs much better documentation

### ISSUE THREAD

**Issue by ldcasillas-progreso** [danthegoodman/gradle-capsule-plugin#15]
The current instructions in the `README.md` simply say to add the following to your `build.gradle` file:

    plugins {
      id "us.kirchmeier.capsule" version "1.0-rc1"
    }

    task fatCapsule(type: FatCapsule) {
      applicationClass 'com.foo.HelloWorld'
    }

But having tried that, these instructions will clearly not work for everybody.  First of all, it doesn't appear that `plugins` configuration block works like this in Gradle 1.12 (what I was using).  Second, there are Gradle rules that restrict where `plugins` may appear.

So basically, I followed the instructions and it didn't work.  And the instructions don't tell me what Gradle version I need in the first place.

**Comment by danthegoodman** [comment]
Fair enough. I wrote in an assumption that people were familiar with Gradle. Given it's complexity, I'm not sure that was a fair expectation to have.

I'll add back in some commentary regarding which Gradle version to use and how to use older versions.

**Comment by danthegoodman** [comment]
@ldcasillas-progreso I tested the plugin with various versions of gradle and found that the plugin requires gradle 2.0 and above. This finding and other instructions have been added to the README.

**Comment by ldcasillas-progreso** [comment]
Thanks!  Yeah, it's better now.
